src/pages/Home.jsx

```diff
@@ -0,0 +1,25 @@
+import img1 from "../assets/beers.png"
+import img2 from "../assets/random-beer.png"
+import img3 from "../assets/new-beer.png"
+
+import { Link } from "react-router-dom"
+function Home() {
+  
+  return (
+    <div>
+        <Link to="/beers"><img src={img1} alt="Image of a balcony with beers" className="page-img"/><br />
+        Home
```

Comment:
Having a `<Link to="/beers">` and the text as "Home" is wrong practice. Pay attention to this type of situations.